tests/pages/books/book/pages/page.js

```diff
@@ -22,7 +22,7 @@ export default create({
   textArea: '[data-test-text-area]',
   isMarked: 'fancy t<mark data-markjs="true">es</mark>t text',
 
-  visitBooks: visitable('/books'),
+  visit: visitable('/books/1/pages/1'),
```

Comment:
`visitFirstPage` ? or event visit(book = 1, page = 1) so it could be customizable?